package.json

```diff
@@ -81,6 +81,7 @@
     "isomorphic-fetch": "2.2.1",
     "lodash": "4.17.4",
     "neon-js": "git+https://github.com/cityofzion/neon-js.git#3.3.3",
+    "node-fetch": "^2.1.1",
```

Comment:
We already have `axios` in this repo -- we should try to avoid bringing in extra packages to prevent bloat.